assets/js/ajax-filters.js

```diff
@@ -1,288 +1,456 @@
 /* global job_manager_ajax_filters */
-jQuery( document ).ready( function ( $ ) {
+jQuery( document ).ready( function( $ ) {
+	var supports_html5_sessionStorage = false;
+	if ( window.localStorage && typeof window.localStorage.setItem === 'function' ) {
+		supports_html5_sessionStorage = true;
+	}
```

Comment:
What if this were a function instead?

```
function supports_html5_session_storage() {
    return window.localStorage && typeof window.localStorage.setItem === 'function';
}
```

Reply:
Changed 943aea9530671045dcf6eec6b8fa4102034a251f